tldw_Server_API/app/services/web_scraping_service.py

```
@@ -491,7 +494,7 @@ async def process_web_scraping_task(
                         except Exception:
                             chunks_for_sql = []
 
-                        media_id, media_uuid, message = db.add_media_with_keywords(
+                        media_id, media_uuid, message = get_media_repository(db).add_media_with_keywords(
```

Ruff (0.15.6)

[warning] Unpacked variable `media_uuid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)